linkedlistutil.cpp

```diff
@@ -0,0 +1,173 @@
+#include <iostream>
+#include <cstdlib>
+#include <stack>
+#include <cmath>
+#include <vector>
+#include <cstring>
+using namespace std;
+
+
+
+typedef struct node
+{
+	int data;
+	struct node * next;
+	node(int x) {
```

Comment:
Ohhh. basically the concept of constructor is instead of the getnewNode function and using calloc, you can do something like
<code> node* newHead = new node(5); </code>